.github/workflows/sorbet.yml

```diff
@@ -52,12 +52,17 @@ jobs:
             git checkout --no-track -B "${BRANCH}" origin/master
           fi
 
-          brew typecheck --update
+          brew typecheck --update --suggest-typed
           if ! git diff --stat --exit-code "Library/Homebrew/sorbet"
           then
             git add "Library/Homebrew/sorbet"
             git commit -m "sorbet: Update RBI files." \
                        -m "Autogenerated by the [sorbet](https://github.com/Homebrew/brew/blob/master/.github/workflows/sorbet.yml) workflow."
+
+            git add "**/*.rb"
```

Comment:
Can this be any more specific? I'd hate to accidentally include files we don't want.

Reply:
Yeh, perhaps limit to `Library/Homebrew` and maybe exclude `vendor` or something?

Another random thought: does this only upgrade and never downgrade sigils?

Reply:
The `vendor` directory is already excluded since Spoom [uses Sorbet's config file](https://github.com/Shopify/spoom#sorbet-configuration-commands).

It appears that it only upgrades and never downgrades, [yes](https://github.com/Shopify/spoom#change-the-sigil-used-in-files).